maths/average_mode.py

```diff
@@ -1,34 +1,29 @@
-def mode(input_list: list) -> list:  # Defining function "mode."
+from typing import Any
+
+
+def mode(input_list: list) -> list[Any]:  # Defining function "mode."

```

Comment:
The comment says exactly what the code says so it can be removed.
```suggestion
def mode(input_list: list) -> list[Any]:
```